packages/ag-charts-community-examples/src/charts-overview/examples/market-index-treemap/main.ts

```diff
@@ -56,36 +56,9 @@ const options: AgChartOptions = {
 }
 
 function tooltipRenderer(params: AgTreemapSeriesTooltipRendererParams<any>) {
-  const { parent, depth, labelKey } = params
-  const customRootText = 'All industries'
-  const title = depth > 1
-    ? parent[labelKey!]
-    : customRootText
-  let content = '<div>'
-  let ellipsis = false
-
-  if (parent) {
-    const maxCount = 5
-    ellipsis = parent.children!.length > maxCount
-    const sorted = parent.children!.slice(0).sort((a: any, b: any) => b.valuation - a.valuation);
-    sorted.slice(0, maxCount).forEach((child: any) => {
-      const bg = child.color || '#272931';
-      content += `<div style="font-weight: bold; color: white; background-color: ${bg
-        }; padding: 5px;"><strong>${child[labelKey!]
-        }</strong>: ${String(
-          isFinite(child.change) ? child.change.toFixed(2) : ''
-        )}%</div>`
-    })
-  }
-  if (ellipsis) {
-    content += `<div style="text-align: center;">...</div>`
-  }
-  content += '</div>'
   return {
-    title,
-    content,
-    backgroundColor: 'gray',
-  }
+    content: `<b>Change</b>: ${(params.datum[params.colorKey!]).toFixed(2)}%`,
```

Comment:
Q: Are we still rendering the same tooltip content after this change? Nice cleanup if so :D

Reply:
Yeah it didn't seem to do anything